contracts/ContractRegistryAccessor.sol

```diff
@@ -10,70 +10,101 @@ import "./spec_interfaces/ISubscriptions.sol";
 import "./spec_interfaces/IDelegation.sol";
 import "./spec_interfaces/IFeesWallet.sol";
 import "./interfaces/IRewards.sol";
-import "./WithClaimableMigrationOwnership.sol";
+import "./WithClaimableRegistryManagement.sol";
 import "./spec_interfaces/IProtocolWallet.sol";
 
-contract ContractRegistryAccessor is WithClaimableMigrationOwnership {
+contract ContractRegistryAccessor is WithClaimableRegistryManagement {
+
+    function isManager(string memory role) internal view returns (bool) {
+        IContractRegistry _contractRegistry = contractRegistry;
+        return msg.sender == registryManager() || _contractRegistry != IContractRegistry(0) && contractRegistry.getManager(role) == msg.sender;
```

Comment:
not updated to the latest change: 
notFinalized and sender = InitManager OR sender = registryManager OR sender = managers(role)
To be updated in another PR